presto-native-execution/presto_cpp/main/http/tests/HttpTest.cpp

```diff
@@ -65,6 +88,23 @@ class HttpServerWrapper {
   folly::Promise<folly::SocketAddress> promise_;
 };
 
+// Make a SSL connection to the server
+class Cb : public folly::AsyncSocket::ConnectCallback {
+ public:
+  explicit Cb(folly::AsyncSSLSocket* sock) : sock_(sock) {}
+  void connectSuccess() noexcept override {
+    success = true;
+    sock_->close();
+  }
+
+  void connectErr(const folly::AsyncSocketException&) noexcept override {
+    success = false;
```

Comment:
What's the success used for? Comment? thanks!
```
s/success/succeeded_/
```

Reply:
The test checks if ssl connection was successful with this variable.

```
  sock->connect(&cb, serverAddress, 1000);
  evb.loop();
  EXPECT_TRUE(cb.success);
  ```